cluster/addons/metadata-proxy/metadata-proxy-configmap.yaml

```diff
@@ -0,0 +1,60 @@
+kind: ConfigMap
```

Comment:
The happy cases should be well tested by feature tests (e.g., logging and monitoring tests should fail in the absence of a metadata server). I intend to push a negative case test after this goes in.